src/test/regress/expected/subquery_in_where.out

```diff
@@ -0,0 +1,679 @@
+-- ===================================================================
+-- test recursive planning functionality with subqueries and CTEs
```

Comment:
`test recursive planning functionality with subqueries and CTEs` maybe mention about subqueries in `WHERE`?